chromium_src/chrome/browser/history/history_utils_unittest.cc

```diff
@@ -33,4 +33,5 @@ TEST(HistoryUtilsTest, VariousURLTest) {
   EXPECT_FALSE(CanAddURLToHistory(GURL("chrome-search://test")));
   EXPECT_FALSE(CanAddURLToHistory(GURL("chrome-distiller://test")));
   EXPECT_FALSE(CanAddURLToHistory(GetSampleLitePageUrl()));
+  EXPECT_FALSE(CanAddURLToHistory(GURL("chrome-extension://odbfpeeihdkbihmopkbjmoonfanlbfcl/home.html"))); // NOLINT
```

Comment:
nit: Usually, we don't use `NOLINT` if we can avoid it.
I think below would be fine.
```
EXPECT_FALSE(CanAddURLToHistory(
    GURL("chrome-extension://odbfpeeihdkbihmopkbjmoonfanlbfcl/home.html")));
```

Reply:
done